octoprint_octorelay/__init__.py

```diff
@@ -202,21 +207,20 @@ def on_shutdown(self):
 
     def get_api_commands(self):
         return {
-            "update": [],
-            "getStatus": [],
-            "listAllStatus":[],
+            "update": ["pin"],
+            "getStatus": ["pin"],
+            "listAllStatus": [],
```

Comment:
yeah, this is a bug, according to the documentation:
https://docs.octoprint.org/en/master/plugins/mixins.html#simpleapiplugin
parameters must be listed here.
But that should be fixed in another PR

Reply:
Created #96 for this

Reply:
PR #97

Reply:
fixed